build/docker/zap-full-scan.py

```diff
@@ -420,6 +424,10 @@ def main(argv):
                 # Save the report
                 write_report(base_dir + report_html, zap.core.htmlreport())
 
+            if report_json:
+                # Save the report
+                write_report(base_dir + report_json, zap.core.jsonreport())
```

Comment:
@claudijd the call `zap.core.jsonreport()` needs to be replaced with `zap._request_other(zap.base_other + 'core/other/jsonreport/')`, for the time being.